plugins/mac-virtualcam/src/obs-plugin/plugin-main.mm

```diff
@@ -108,8 +108,20 @@ - (void)request:(nonnull OSSystemExtensionRequest *)request didFailWithError:(no
 
 - (void)request:(nonnull OSSystemExtensionRequest *)request didFinishWithResult:(OSSystemExtensionRequestResult)result
 {
-    self.installed = YES;
-    blog(LOG_INFO, "macOS Camera Extension activated successfully.");
+    switch (result) {
+        case OSSystemExtensionRequestCompleted:
+            self.installed = YES;
+            blog(LOG_INFO, "macOS Camera Extension activated successfully.");
+            break;
+        case OSSystemExtensionRequestWillCompleteAfterReboot:
+            self.lastErrorMessage =
+                [NSString stringWithUTF8String:obs_module_text("Error.SystemExtension.CompleteAfterReboot")];
+            blog(LOG_INFO, "macOS Camera Extension will activate after reboot.");
+            break;
+        default:
+            blog(LOG_INFO, "macOS Camera Extension activation finished with unknown result %ld", (long) result);
```

Comment:
I don't think we need a `default` branch here because the switch statement should be exhaustive just with the two values checked above.